src/lib/api.js

```diff
@@ -1,77 +1,114 @@
 import axios from 'axios';
 
-const api = {
-    /**
-     * Filter empty values from the Axios request options object
-     *
-     * @private
-     * @param {Object} options - The request options
-     * @return {Object} The cleaned request options
-     */
-    filterOptions(options = {}) {
-        const result = {};
+/**
+ * Retrieves JSON from response.
+ *
+ * @private
+ * @param {Response} response - Response to parse
+ * @return {Promise|Response} Response if 204 or 202, otherwise promise that resolves with JSON
+ */
+export const parseResponse = (response) => {
```

Comment:
Is this used outside of the tests? If not, can we avoid exporting and test its consuming methods, instead?

Reply:
Yeah, it is only used in the preview tests.